GetLatestMigrationsStatus/handler.ts

```diff
@@ -65,21 +86,51 @@ export const createSqlStatus = (dbConfig: IDecodableConfigPostgreSQL) => (
     .as("t")
     .toQuery() as NonEmptyString;
 
-export const createHandler = (): Handler => (): ReturnType<Handler> =>
+export const getLatestMigrationByOrganizationFiscalCode = (
```

Comment:
Please add a comment line about what these small functions do, it helps to read the overall procedure